folly/String.h

```diff
@@ -185,9 +185,12 @@ std::string& stringAppendf(
  * As with vsnprintf() itself, the value of ap is undefined after the call.
  * These functions do not call va_end() on ap.
  */
-std::string stringVPrintf(const char* format, va_list ap);
-void stringVPrintf(std::string* out, const char* format, va_list ap);
-std::string& stringVAppendf(std::string* out, const char* format, va_list ap);
+std::string stringVPrintf(const char* format, va_list ap)
```

Comment:
General style - attributes should precede the signature. Here and elsewhere.

Reply:
I only matched the way it already was.